Sources/DependencyClients/TmuxService.swift

```diff
@@ -0,0 +1,206 @@
+import Foundation
+import Models
+
+struct TmuxService: Sendable {
+  private let config: Models.SSHServerConfiguration
+  private let sshClient: SSHClient
+
+  init(config: Models.SSHServerConfiguration) {
+    self.config = config
+    self.sshClient = SSHClient()
```

Comment:
I don't understand, does this create a new ssh client for each tmux service?
I think that the tmux service should receive ssh client via ctor args